crates/sqlexec/src/planner/session_planner.rs

```diff
@@ -2057,10 +2066,13 @@ fn convert_simple_data_type(sql_type: &ast::DataType) -> Result<DataType> {
             // Explicitly list all other types so that if sqlparser
             // adds/changes the `ast::DataType` the compiler will tell us on upgrade
             // and avoid bugs like https://github.com/apache/arrow-datafusion/issues/3059
-            ast::DataType::Nvarchar(_)
+            ast::DataType::Int64
+            | ast::DataType::Float64
+            | ast::DataType::Nvarchar(_)
             | ast::DataType::JSON
             | ast::DataType::Uuid
             | ast::DataType::Binary(_)
+            | ast::DataType::Bytes(_)
```

Comment:
Do we want to add these datatypes? I excluded them since `Binary` was already excluded…